spec/requests/locations_spec.rb

```
@@ -21,70 +27,110 @@
   #Needs to verify user owns location
   it 'will return a location based on id passed ' do
 
     get "/api/v1/locations/#{location.id}", headers: {"ACCEPT" => "application/json",  "Token" => "1234"}
-
-      expect(response).to have_http_status(200)
+    
       parsed_json = JSON.parse(response.body)
       expect(parsed_json['location']['street']).to eq('1200 front st.')
       expect(parsed_json['location']['city']).to eq('Durham')
       expect(parsed_json['location']['state']).to eq('NC')
       expect(parsed_json['location']['zip']).to eq( "27705")
-  end
-  #Delete Record based on id
-  it 'will delete a location based on id passed ' do
-
-    delete "/api/v1/locations/#{location.id}", headers: {"ACCEPT" => "application/json",  "Token" => "1234"}
-
       expect(response).to have_http_status(200)
-      parsed_json = JSON.parse(response.body)
-      expect(parsed_json['success']).to eq(true)
   end
-
+ 
   it 'will update a location related to logged in user' do
 
     put "/api/v1/locations/#{location.id}", headers: {"ACCEPT" => "application/json",  "Token" => "1234"},
-      params: {location:{street:"210 Front Street",
+      params: {location:{street:"2210 Front Street",
       city:"Burlington",
       state:"NC",
-      zip: "27217"}}
+      zip: "27215"}}
 
       expect(response).to have_http_status(200)
       parsed_json = JSON.parse(response.body)
-      expect(parsed_json['location']['street']).to eq('210 Front Street')
+      expect(parsed_json['location']['street']).to eq('2210 Front Street')
       expect(parsed_json['location']['city']).to eq("Burlington")
       expect(parsed_json['location']['state']).to eq('NC')
-      expect(parsed_json['location']['zip']).to eq( "27217")
+      expect(parsed_json['location']['zip']).to eq( "27215")
       expect(parsed_json['location']['location_relationships'][0]['driver_id']).to eq( driver.id)
   end
+  
+   it 'Should return a error message.' do
```

This is not a good it statement.  You want a statement that says Updating a location with bad values should return an error message.